port/win32/omrfile.c

```diff
@@ -101,7 +99,7 @@ fromHandle(OMRPortLibrary *portLibrary, HANDLE handle)
  * 					If the return value is not the same as @ref unicodeBuffer, file_get_unicode_path had to allocate memory to hold the
  * 					unicode path - the caller is responsible for freeing this memory by calling @ref omrmem_free_memory on the returned value.
  */
-static wchar_t *
+wchar_t *
```

Comment:
since this is going to be exported can you prepend `port_` to the function name?

Reply:
Renamed to file_get_unicode_path() to port_file_get_unicode_path().